src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -715,4 +715,7 @@
   <data name="InfoBarDismissButton.Content" xml:space="preserve">
     <value>Don't show again</value>
   </data>
+  <data name="ElevationShield.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>This Terminal window is running as an Administrator</value>

```

Comment:
Wording here feels off... but I cannot figure out why.,

Reply:
* "This Terminal window is running with elevated permissions"?
* "This Terminal window is running in High-IL"
* "This Terminal window is running as Admin"
* "This Terminal window is running elevated"
* "This Terminal window has elevated permissions"

Reply:
I'm fond of this one.

```suggestion
    <value>This Terminal window is running as Admin</value>
```